sig-pm/README.md

```diff
@@ -2,7 +2,7 @@
 
 ### Focus
 
-The Kubernetes PM Group is focused on a variety of challenges: from the qualification and successful management of user requests, one of the many tasks of product management, through the continued improvement of the processes used by the Kubernetes community to maintain the Kubernetes Project itself which more closely relates to traditional project or program management. Besides helping to discover both what to build and how to build it, the PM Group also helps to try and keep the wheels on this spaceship we are all building together; bringing together people who think about Kubernetes as both a vibrant community of humans and technical program is another primary focus of this group. Members of the Kubernetes PM Group can assume [certain additional](https://github.com/kubernetes/community/blob/master/project-managers/README.md) responsibilities to help maintain the Kubernetes Project itself. It is also important to remember that the role of managing an open source project is very new and largely unscoped for a project as large as Kubernetes; we are learning too and we are excited to learn how we can best serve the community of users and contributors.
+The Kubernetes PM Group focuses on aspects of product management, such as the qualification and successful management of user requests, and aspects of project and program management such as he continued improvement of the processes used by the Kubernetes community to maintain the Kubernetes Project itself. Besides helping to discover both what to build and how to build it, the PM Group also helps to try and keep the wheels on this spaceship we are all building together; bringing together people who think about Kubernetes as both a vibrant community of humans and technical program is another primary focus of this group. Members of the Kubernetes PM Group can assume [certain additional](https://github.com/kubernetes/community/blob/master/project-managers/README.md) responsibilities to help maintain the Kubernetes Project itself. It is also important to remember that the role of managing an open source project is very new and largely unscoped for a project as large as Kubernetes; we are learning too and we are excited to learn how we can best serve the community of users and contributors.
```

Comment:
Thanks for the suggestion @stp-ip but I think I'll keep the copy largely unchanged because I was aiming for `Besides helping to discover both what to build and how to build it` to relate to the previous sentence talking about Product/Project/Program Management rather than standing as it's own thought point and I wanted that thought connected to the thought of the SIG as a rallying point for people such that  `the PM Group also helps to try and keep the wheels on this spaceship we are all building together` and `bringing together people who think about Kubernetes as both a vibrant community of humans and technical program is another primary focus of this group` formed a single sentence. It's a bit wordy but I think generally ok...at least good enough to merge